pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -140,11 +140,8 @@ in ''
 
      set -e
      if [[ -n "$(ls target/build/${crateName}.out)" ]]; then
```

Comment:
Don't nest two `if` with a single clause each. You can chain conditions inside `[[` with `&&`.

Reply:
I’m not super confident dealing with the mess that bash is to start refactoring this code.

Reply:
```sh
if [[ -n "$(ls target/build/${crateName}.out)" && -e "${libPath}" ]]; then
  cp -r target/build/${crateName}.out/* $(dirname ${libPath}) #*/
fi
```

Basically.

Although there's some quoting I could be fussy about, that seems off topic.

Reply:
Please make sure I didn't accidentally copy paste the wrong thing :smile: